styles/DTDDocStyle.css

```diff
@@ -0,0 +1,57 @@
+body { background-color:white; font-family: Verdana, Geneva, Arial, Helvetica, sans-serif; font-size: small; }
```

Comment:
All the CSS files belong under tools/styles/${lp.style.edition} either directly or in a subdirectory, as in the case of taglibs/stylesheet.css (which needs to keep that specific name).

And the naming convention is not consistent with Liferay's CSS file name convention. Please refer to existing CSS files.